examples/ecs-ec2/variables.tf

```diff
@@ -0,0 +1,44 @@
+variable "ecs_cluster_name" {
+  description = "Name of the AWS ECS Cluster to deploy the Coralogix OTEL Collector. Supports Amazon EC2 instances only, not Fargate."
+  type        = string
+}
+
+variable "image_version" {
+  description = "The Coralogix Open Telemetry Distribution Image Version/Tag. Defaults to \"latest\". See: https://hub.docker.com/r/coralogixrepo/coralogix-otel-collector/tags"
+  type        = string
+  default     = "latest"
+}
+
+variable "memory" {
+  description = "The amount of memory (in MiB) used by the task. Note that your cluster must have sufficient memory available to support the given value."
+  type        = number
+  default     = 256
+}
+
+variable "coralogix_region" {
```

Comment:
Need to support also custom region

Reply:
Will test across multiple regions.

Reply:
The custom region is basically address that is not in the mapping list
It is for use cases of private cluster or new cluster ....
So we need to be able to put the coralogix endpoint

Reply:
```custom_domain``` parameter added.